ReactAndroid/src/main/jni/react/jni/OnLoad.cpp

```diff
@@ -84,6 +85,7 @@ extern "C" JNIEXPORT jint JNI_OnLoad(JavaVM *vm, void *reserved) {
 #ifdef WITH_INSPECTOR
     JInspector::registerNatives();
 #endif
+    JReactMarker::setLogPerfMarkerIfNeeded();
```

Comment:
Should the other calls from jscexecutor / reactexecutor OnLoad now be removed? This will always be ran before them.

Reply:
done